kumascript/macros/MDNSidebar.ejs

```diff
@@ -6,244 +6,699 @@ const baseURL = `/${locale}/docs/MDN/`;
+        "getting_started": "Getting started with MDN Web Docs",
+        "our_repositories": "MDN Web Docs Repositories",
+        "translated_content": "MDN Web Docs Localization",
+        "security_vulnerability_response": "Security vulnerability response steps",
+      "open_source_etiquette": "Open source etiquette",
+      "communication_channels": "Communication channels",
+      "discussions": "Community discussions",
+      "learn_forum": "Learn forum",
+      "issues": "Issue guidelines",
+      "prs": "Pull request guidelines",
+      "mdn_content": "Contributing to MDN Web Docs",
+        "contributing_mdn_content": "Contributing to MDN Web Docs content",
+        "contributing_mdn_content_issues": "Using issues on MDN Web Docs",
+        "contributing_mdn_content_prs": "Pull request etiquette and process for MDN Web Docs",
+      "roles_and_teams": "MDN Web Docs roles and teams",
+    "contributing_to_mdn": "Contributing to MDN",
+      "contributing_to_mdn_overview": "Overview",
+      "documentation_processes": "Documentation processes",
+      "mdn_workstream_assessment": "MDN Workstream assessment and project setup process",
+    "writing_guide": "Writing guidelines",
+      "write_overview": "Overview",
+      "what_we_write": "What we write",
+      "writing_style_guide": "Writing style guide",
+      "how_to_guides": "How-to guides",
+         "how_to_guides_overview": "Overview",
+         "how_to_images_media": "How to add images and media",
+         "how_to_crud_pages": "How to create, move, delete, and edit pages",
+         "how_to_css_property": "How to document a CSS property",
+         "how_to_http_header": "How to document an HTTP header",
+         "how_to_web_errors": "How to document web errors",
+         "how_to_research": "How to research a technology",
+         "how_to_use_structured_data": "How to use structured data",
+         "how_to_api_ref": "How to write an API reference",
+         "how_to_glossary": "How to write an entry in the glossary",
+         "how_to_markdown": "How to write in Markdown",
+      "page_structures": "Page structures",
+        "page_structures_overview": "Overview",
+        "page_structures_banners_notices": "Banners and notices",
+        "page_structures_code_examples": "Code examples",
+        "page_structures_compatibility_tables": "Compatibility tables and the browser compatibility data repository (BCD)",
+        "page_structures_live_examples": "Live examples",
+        "page_structures_page_types": "Page types",
+        "page_structures_quicklinks": "Sidebars and other link macros",
+        "page_structures_specification_tables": "Specification tables",
+        "page_structures_syntax_sections": "Syntax sections",
+        "page_structures_using_macros": "Using macros",
+      "attribution": "Attribution and copyright",
+      "statuses": "Experimental, deprecated and obsolete",
+    "changelog": "Changelog",
+    "kitchensink": "The MDN Content Kitchensink"
```

Comment:
I don't think we should list this one. It is a technical page used to test MDN.

Reply:
It could serve as a cheatsheet of all the available constructs. WDYT?

Reply:
I don't think so: it is not maintained with this goal. And I'm not convinced this additional goal will not conflict with its current tecnhical goal.